### PR TITLE
v0.9 release branch: Namespace changes and fixes for LCG_92 compatibility

### PR DESCRIPTION
**Pull request by vvolkl**
* Major namespace changes in DD4hep
* Minor changes in Gaudi, ACTS ...
* Rename GridPhiEta to FCCSWGridPhiEta to avoid broken duplicate in DD4hep

### CODE REVIEW

**Comment by vvolkl** [comment]
Remaining todo:
[ ] change `init.sh` to use cvmfs installation

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,5 +1,7 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 2.8.5)
 
+set(LCG_releases_base  /cvmfs/sft.cern.ch/lcg/releases)
```
[the remark this answers is not in the document]
Comment (vvolkl) [reply CMakeLists.txt]: No, I guess this is a remnant of my installing an experimental stack for v0.9. Good catch, I removed it! :)

File: Detector/DetExtensions/DetExtensions/DetCylinderVolume.h [hunk 2]
```diff
@@ -11,7 +11,7 @@
 
 #include "DetExtensions/IDetExtension.h"
 
-namespace DD4hep {
+namespace dd4hep {
 namespace Geometry {
```
[the remark this answers is not in the document]
Comment (vvolkl) [reply Detector/DetExtensions/DetExtensions/DetCylinderVolume.h]: Ah, it seems to me this whole subdirectory is obsolete, this is why there was no compilation error. I'll just clean up the whole thing.

File: Reconstruction/RecCalorimeter/src/components/SingleCaloTowerTool.cpp [hunk 3]
```diff
@@ -0,0 +1,183 @@
+#include "SingleCaloTowerTool.h"
```
Comment (zaborowska) [inline Reconstruction/RecCalorimeter/src/components/SingleCaloTowerTool.cpp]: this file probably was removed by some previous PR ? (Hence it appears as if you add it?)

File: Reconstruction/RecCalorimeter/src/components/SingleCaloTowerTool.h [hunk 4]
```diff
@@ -0,0 +1,144 @@
+#ifndef RECCALORIMETER_SINGLECALOTOWERTOOL_H
```
Comment (zaborowska) [inline Reconstruction/RecCalorimeter/src/components/SingleCaloTowerTool.h]: same here

File: Test/TestGeometry/CMakeLists.txt [hunk 5]
```diff
@@ -60,11 +50,11 @@ gaudi_add_test(TestGFlashHomoCalo
...
 # Test the detector baselines:
-gaudi_add_test(OverlapCheckExtrusionsMaterialFCChh
-               WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}
-               COMMAND python ${DD4hep_ROOT}/python/checkOverlaps.py -c Detector/DetFCChhBaseline1/compact/FCChh_DectMasterMaterial.xml
-               PASSREGEX "Number of illegal overlaps/extrusions : 0" )
-gaudi_add_test(OverlapCheckSamplingMaterialFCChh
-               WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}
-               COMMAND python ${DD4hep_ROOT}/python/checkOverlaps.py -c Detector/DetFCChhBaseline1/compact/FCChh_DectMasterMaterial.xml -o s
-               PASSREGEX "Number of illegal overlaps/extrusions : 0" )
+#gaudi_add_test(OverlapCheckExtrusionsMaterialFCChh
```
Comment (zaborowska) [inline Test/TestGeometry/CMakeLists.txt]: if it doesn't exist any more probably we should delete it

File: Test/TestGeometry/tests/options/geant_overlap_test_fcchh.py [hunk 6]
```diff
@@ -10,7 +10,7 @@
                         detector='SimG4DD4hepDetector',
                         physicslist="SimG4FtfpBert",
                         actions="SimG4FullSimActions",
-                        g4PostInitCommands=['/geometry/test/recursion_depth 3',  '/geometry/test/run'])
+                        g4PostInitCommands=['/geometry/test/recursion_depth 2',  '/geometry/test/run'])
```
Comment (zaborowska) [inline Test/TestGeometry/tests/options/geant_overlap_test_fcchh.py]: why this change? time?

Comment (vvolkl) [reply Test/TestGeometry/tests/options/geant_overlap_test_fcchh.py]: yes, this test was taking more than 20 min like this. I can change it back, though!

Comment (zaborowska) [reply Test/TestGeometry/tests/options/geant_overlap_test_fcchh.py]: But did you check it is useful with this depth? Maybe we could run it only in nightlies or sth? @JavierCVilla , what do you think?
